docker/build/docker.ubuntu.Dockerfile

```diff
@@ -0,0 +1,19 @@
+ARG ARCH="amd64"
+
+FROM ${ARCH}/ubuntu:18.04
+ENV GOLANG_VERSION 1.12 \
+    GOROOT=/usr/local/go \
+    GOPATH=$HOME/go \
+    PATH=$GOPATH/bin:$GOROOT/bin:$PATH
+RUN apt-get update && apt-get install curl wget python build-essential apt-transport-https ca-certificates software-properties-common -y && \
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - && \
+    add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" && \
+    apt-get update && apt-get install docker-ce -y
+WORKDIR /root
+RUN wget https://dl.google.com/go/go1.12.linux-amd64.tar.gz \
+    && tar -xvf go1.12.linux-amd64.tar.gz && \
+    mv go /usr/local
```

Comment:
Could you move the go installation into the package.sh script, ( or a different script called by package.sh ) ?
I'd like to minimize the number of location we're doing go installations, and eventually get to a single script.

Reply:
Will change.

Reply:
I like that, it's much cleaner.  We'll have to remove those same lines from the other dockerfiles in that dir as well.